cget/prefix.py

```diff
@@ -183,7 +183,7 @@ def get_builder_path(self, *paths):
     @contextlib.contextmanager
     def create_builder(self, name, tmp=False):
         pre = ''
-        if tmp: pre = 'tmp-'
+        if tmp: pre = '/tmp/cgettmp-'
```

Comment:
I dont think this prefix needs to change. This only used in the build directory.

Reply:
By forcing the temp files to be consistent we ensure that even though the prefix would be different `ccache` sees the same path and can reuse the build artifacts. 

This is indeed a temporary directory since in the same function it is being removed, so the exact location of this directory does not matter since it is not supposed to persist.

I would however, change the code so that its cross platform.

Reply:
Gotcha, will update the branch.

Reply:
Updated